pkg/portal/portal.go

```diff
@@ -284,14 +284,14 @@ func (p *portal) aadAuthenticatedRoutes(r *mux.Router) {
 		regexp, _ := regexp.Compile(`v[1,2]/build/.*\..*`)
 		name := regexp.FindString(name)
 		switch name {
-		case "v1/build/index.html":
-			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
 		case "v2/build/index.html":
-			r.NewRoute().Methods(http.MethodGet).Path("/v2").HandlerFunc(p.indexV2)
+			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.indexV2)
+		case "v1/build/index.html":
+			r.NewRoute().Methods(http.MethodGet).Path("/v1").HandlerFunc(p.index)
 		case "":
 		default:
-			fmtName := strings.TrimPrefix(name, "v1/build/")
-			fmtName = strings.TrimPrefix(fmtName, "v2/build/")
+			fmtName := strings.TrimPrefix(name, "v2/build/")
+			fmtName = strings.TrimPrefix(fmtName, "v1/build/")
```

Comment:
do we need this change? because AFAIK the url isn't `v1/build/v2/build` ? it was just to make sure we removed all occurrences of `v1/build/` and `v2/build`